### ISSUE TITLE
Consider a place to host shared telemetry documentation

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/telemetry-android#67]
I'm starting to see our docs sprawl out and starting to duplicate each other. I'm concerned that when things change, we won't update them all in the right places.

I propose we do some part of:
- Remove all duplication of docs: implementation details about event pings should be hosted in one place such as this library's wiki (it's duplicated in the Focus and Firefox TV wikis atm)
- Centralize *all* telemetry documentation on one page (mozilla-mobile.github.io/telemetry?). This way we'll know all of the telemetry docs that exist for all the products that use this library, which will help us prevent duplication and be more likely to keep everything up-to-date.

Note that we may need to version the documentation just in case a product isn't using the latest library.

---

### Example of sprawl
For the docs I know about, currently we have:
- [core ping in mozilla-central](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/core-ping.html)
- [Firefox Focus Android docs](https://github.com/mozilla-mobile/focus-android/wiki/Telemetry)
- [Firefox TV docs](https://github.com/mozilla-mobile/firefox-tv/wiki/Telemetry)

Some of the documentation is shared between the last two which have implementation notes about the event pings (at the bottom of their wikis). This implementation actually comes from the library and should be hosted somewhere with the library (or in some central location).

CC @pocmo @liuche @Sdaswani

**Comment by Sdaswani** [comment]
I agree this is a starting to get disjoint so I think consolidation makes sense. Alternatively it would be nice if the implementation generated the documentation automatically. It wouldn’t be centralized in that case but at least it would be in a standard location and would auto-update.

**Comment by mcomella** [comment]
> Alternatively it would be nice if the implementation generated the documentation automatically.

In it's simplest form, we could generate the documentation from the javadoc.
